web/static/main.js

```
@@ -3919,37 +3928,69 @@ function stopCurrentTTS() {
   }
 
-  // Reset chunked processing state
   window.currentChunkIndex = 0;
   window.totalChunks = 0;
   window.isProcessingChunks = false;
+  window.currentUtteranceAllChunksReceived = false;
+  window.ttsUtteranceQueue = [];
 
-  // Clear pending chunks and timeouts
   window.pendingChunks.clear();
   window.chunkTimeouts.forEach(timeoutId => clearTimeout(timeoutId));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)